build/chip/fallback_lkgt

```diff
@@ -0,0 +1 @@
+750275722
```

Comment:
Can we have a more human readable name (even if long) than `lkgt`  ? guessing last_known_good_time however that will not be obvious to others later on.

Reply:
done